activiti-api-task-runtime/src/main/java/org/activiti/api/task/runtime/TaskRuntime.java

```diff
@@ -113,10 +115,13 @@
      * - this method returns a shallow Task with the necessary information to validate that the task was deleted
      */
     Task delete(DeleteTaskPayload deleteTaskPayload);
+    
+    //Old method: check if it should be removed
+    void setVariables(SetTaskVariablesPayload setTaskVariablesPayload);
```

Comment:
@CTI777 @erdemedeiros my opinions is that we ended up with this for legacy reasons, and we can clearly document why we are changing that API.
It is also something to note that "deleteVariable" doesn't exist.. for tasks that might be an option to think about and to document.

Reply:
@salaboy yes, it makes sense.